app/components/object-inspector/sort-properties.js

```diff
@@ -21,7 +46,15 @@ export default Component.extend({
      * @property sortProperties
      * @type {Array<String>}
      */
-    this.sortProperties = ['name'];
+    this.sortProperties = [
+      'isFunction',
```

Comment:
I'm curious, how did you decide the ordering for sorting?

Reply:
mmm, a mix between how it looks like (most times) in the code and whats useful:
1. services are always at the top
2. then comes properties, tracked
3. later computed and getters,   -> but I think computed/getters and tracked are more interesting than normal properties, so there are more at the top
4. finally functions at the bottom (sometimes useful if one wants to debug the function, then `send to console` and click on the function and chrome will send you directly to the source code) (i would even prefer if there was a `goto` for functions instead of `send to console` :) )

So, final order is:
Service -> Tracked -> Computed -> Getter -> Property -> Function

the `sortProperties` is more a short trial & error :)
had to put `isFunction` at the top, since its also a `isProperty`

Reply:
mmm, i think this is what I had initially too.
But for arrays i wanted to see the computed properties like `lastObject` & `firstObject` at the beginning and not scroll down 100 items...
But this is only in case of arrays and 2 CPs :) . 
So, if you prefer the order similar to eslint-plugin-ember, i go ahead and change it

Reply:
I think maybe we should follow eslint-plugin-emebr, but you make a good point, so perhaps for arrays we could have a special case that changes the order?